ui/tests/acceptance/job-services-test.js

```diff
@@ -0,0 +1,54 @@
+import { module, test } from 'qunit';
+import { find, findAll, currentURL, settled } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { allScenarios } from '../../mirage/scenarios/default';
+
+import a11yAudit from 'nomad-ui/tests/helpers/a11y-audit';
+import Services from 'nomad-ui/tests/pages/jobs/job/services';
+
+module('Acceptance | job services', function (hooks) {
```

Comment:
`question`: Have you run this test a few times? I've noticed that this has been flaky.

Reply:
I've only encountered flakiness when running other tests against its test scenario; I'll keep monitoring it and isolate the scenario within the test itself if it keeps flaking out.